docker/test/performance-comparison/compare.sh

```diff
@@ -7,6 +7,10 @@ trap 'kill $(jobs -pr) ||:' EXIT
 stage=${stage:-}
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# https://github.com/jemalloc/jemalloc/wiki/Getting-Started
+export MALLOC_CONF="percpu_arena:disabled"
+ln -s "percpu_arena:disabled" /etc/malloc.conf
```

Comment:
```suggestion
ln -s "confirm_conf:true,percpu_arena:disabled" /etc/malloc.conf
```

I would also suggest adding `confirm_conf:true`, it will print all options with values:

```
$ MALLOC_CONF=confirm_conf:true,percpu_arena:disabled ./clickhouse server -V
<jemalloc>: malloc_conf #1 (string specified via --with-malloc-conf): "percpu_arena:percpu,oversize_threshold:0,muzzy_decay_ms:10000"
<jemalloc>: -- Set conf value: percpu_arena:percpu
<jemalloc>: -- Set conf value: oversize_threshold:0
<jemalloc>: -- Set conf value: muzzy_decay_ms:10000
<jemalloc>: malloc_conf #2 (string pointed to by the global variable malloc_conf): ""
<jemalloc>: malloc_conf #3 ("name" of the file referenced by the symbolic link named /etc/malloc.conf): "confirm_conf:true"
<jemalloc>: -- Set conf value: confirm_conf:true
<jemalloc>: malloc_conf #4 (value of the environment variable MALLOC_CONF): "confirm_conf:true,percpu_arena:disabled"
<jemalloc>: -- Set conf value: confirm_conf:true
<jemalloc>: -- Set conf value: percpu_arena:disabled
ClickHouse server version 20.10.1.4618.
```